spec/models/va_profile_redis/contact_information_v2_spec.rb

```diff
@@ -0,0 +1,336 @@
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.residential_address).to eq residence
+            expect(contact_info.residential_address.class).to eq VAProfile::Models::V2::Address
+          end
+        end
+      end
+
+      describe '#mailing_address' do
+        it 'returns the users mailing address object', :aggregate_failures do
+          residence = address_for VAProfile::Models::V2::Address::CORRESPONDENCE
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.mailing_address).to eq residence
+            expect(contact_info.mailing_address.class).to eq VAProfile::Models::V2::Address
+          end
+        end
+      end
+
+      describe '#home_phone' do
+        it 'returns the users home phone object', :aggregate_failures do
+          phone = phone_for VAProfile::Models::Telephone::HOME
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.home_phone).to eq phone
+            expect(contact_info.home_phone.class).to eq VAProfile::Models::Telephone
+          end
+        end
+      end
+
+      describe '#mobile_phone' do
+        it 'returns the users mobile phone object', :aggregate_failures do
+          phone = phone_for VAProfile::Models::Telephone::MOBILE
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.mobile_phone).to eq phone
+            expect(contact_info.mobile_phone.class).to eq VAProfile::Models::Telephone
+          end
+        end
+      end
+
+      describe '#work_phone' do
+        it 'returns the users work phone object', :aggregate_failures do
+          phone = phone_for VAProfile::Models::Telephone::WORK
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.work_phone).to eq phone
+            expect(contact_info.work_phone.class).to eq VAProfile::Models::Telephone
+          end
+        end
+      end
+
+      describe '#temporary_phone' do
+        it 'returns the users temporary phone object', :aggregate_failures do
+          phone = phone_for VAProfile::Models::Telephone::TEMPORARY
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.temporary_phone).to eq phone
+            expect(contact_info.temporary_phone.class).to eq VAProfile::Models::Telephone
+          end
+        end
+      end
+
+      describe '#fax_number' do
+        it 'returns the users FAX object', :aggregate_failures do
+          phone = phone_for VAProfile::Models::Telephone::FAX
+          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
+            expect(contact_info.fax_number).to eq phone
```

Comment:
This could be called `fax` to distinguish it from the other phone numbers. 
```suggestion
          fax = phone_for VAProfile::Models::Telephone::FAX
          VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
            expect(contact_info.fax_number).to eq fax
```